client/src/infographic/Infographic.js

```diff
@@ -422,8 +422,19 @@ const Infographic = ({
       <Redirect to={infograph_href_template(subject_parent, bubble_id, "/")} />
     );
   }
-
   const title = text_maker("infographic_for", { subject });
+  const breadcrumb_program =
```

Comment:
As far as I can see, you can directly pass in array of JSX elements. So you can pass in the `<a>` tags directly. Please remove the changes in NavComponents and try passing in react elements directly